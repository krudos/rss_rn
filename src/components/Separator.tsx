import {StyleSheet, View} from 'react-native';
import React from 'react';

export const Separator = () => {
  return <View style={styles.container} />;
};

const styles = StyleSheet.create({
  container: {
    height: 1,
    backgroundColor: 'gray',
  },
});
