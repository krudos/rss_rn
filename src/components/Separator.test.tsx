import React from 'react';
import {render} from '@testing-library/react-native';
import {Separator} from './Separator';
describe('Separator', () => {
  test('renders correctly', () => {
    const {toJSON} = render(<Separator />);
    expect(toJSON()).toMatchSnapshot();
  });
});
