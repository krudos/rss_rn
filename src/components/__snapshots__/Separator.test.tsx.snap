// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "gray",
      "height": 1,
    }
  }
/>
`;
